### PR TITLE
Style guide nav revamp

### PR DESCRIPTION
**Pull request by mmfred**
working on implementing the nav proposed by the style guide group.

### CODE REVIEW

File: src/nav/style-guide.yml [hunk 1]
```diff
@@ -1,119 +1,128 @@
...
-        path: /docs/style-guide/article-templates/landing-page-template
-      - title: Agent release notes template 1.2.3
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
+    - title: Writing great content at New Relic
+      path: /docs/style-guide/writing-strategies
+      pages: 
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
```
Comment (barbnewrelic) [inline src/nav/style-guide.yml:10]: I would love to see the 5Qs doc in the "Writing great content" section but defer to you,

File: src/nav/style-guide.yml [hunk 2]
```diff
@@ -1,119 +1,128 @@
...
+          path: /docs/style-guide/formatting/formatting-terminal-commands
+        - title: Titles, short titles, and paths
+          path: /docs/style-guide/formatting/titles
+        - title: UI paths
+          path: /docs/style-guide/formatting/ui-paths
+    - title: Images, videos, and captions
+      path: /docs/style-guide/images
+      pages:
+        - title: Screenshots and images
+          path: /docs/style-guide/images/screenshots-images
```
Comment (barbnewrelic) [inline src/nav/style-guide.yml:49]: I don't understand why the standalone "Embed images" and "Embed videos" docs are not part of the "Images, videos, and captions" category

File: src/nav/style-guide.yml [hunk 3]
```diff
@@ -1,119 +1,128 @@
...
-        path: /docs/style-guide/article-templates/agent-api-guide-template
-      - title: apiStyleGuidelines (Example agent API)
-        path: /docs/style-guide/article-templates/apistyleguidelines-example-agent-api
-      - title: GraphQL API tutorial template
-        path: /docs/style-guide/article-templates/graphql-api-tutorial-template
-      - title: Landing page template
-        path: /docs/style-guide/article-templates/landing-page-template
-      - title: Agent release notes template 1.2.3
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
+    - title: Writing great content at New Relic
```
Comment (barbnewrelic) [inline src/nav/style-guide.yml:4]: Overall I think this is fantastic. Great job!

File: src/nav/style-guide.yml [hunk 4]
```diff
@@ -1,119 +1,128 @@
...
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
+    - title: Capitalization
+      path: /docs/style-guide/capitalization
+      pages:
+        - title: Capitalization
+          path: /docs/style-guide/capitalization/capitalization
+    - title: Formatting
```
Comment (paperclypse) [inline src/nav/style-guide.yml:16]: There's a lot of content in "Formatting". Maybe it's worth considering breaking it up into slightly more granular categories?

Just a loose, back-of-the-napkin suggestion:

+ Text formatting (could include hyperlinks and headings)
+ Code formatting
+ UI formatting
+ Lists and tables (could include collapsers)

File: src/nav/style-guide.yml (hunk 2 above)
Comment (urbiz-nr) [reply src/nav/style-guide.yml:49]: Coming to say the same thing.

File: src/nav/style-guide.yml [hunk 5]
```diff
@@ -1,119 +1,128 @@
...
+    - title: Writing docs
+      path: /docs/style-guide/writing-docs
+      pages:
+        - title: Five questions
+          path: /docs/style-guide/writing-docs/five-questions-help-write-docs
+        - title: Organize your docs page
+          path: /docs/style-guide/writing-docs/organize-doc
+        - title: Docs translation
+          path: /docs/style-guide/writing-docs/docs-translation
+        - title: How to write docs
```
Comment (urbiz-nr) [inline src/nav/style-guide.yml:68]: Are the docs here too hidden?

File: src/nav/style-guide.yml (hunk 3 above)
Comment (urbiz-nr) [reply src/nav/style-guide.yml:4]: +1. This is great!

File: src/nav/style-guide.yml (hunk 1 above)
Comment (mmfred) [reply src/nav/style-guide.yml:10]: I have that thought as well. I was hoping to move it up after a review for docs only info, but there are other places where I haven't been able to do that yet, so why wait on this one! Thanks.

File: src/nav/style-guide.yml (hunk 5 above)
Comment (mmfred) [reply src/nav/style-guide.yml:68]: @urbiz-nr - can you say a little more about this concern? Some things can still move around, and will.

File: src/nav/style-guide.yml (hunk 4 above)
Comment (barbnewrelic) [reply src/nav/style-guide.yml:16]: I like this idea! Maybe a separate category about structure, which could address things like headings and subheadings, length of page, use of lists, tables, collapsers

Comment (paperclypse) [reply src/nav/style-guide.yml:16]: @barbnewrelic Structure-related docs is great! I was struggling to come up with a better category. It makes a lot of sense to divide things between structure and formatting.

File: src/nav/style-guide.yml (hunk 5 above)
Comment (urbiz-nr) [reply src/nav/style-guide.yml:68]: I feel like "basic" stuff is easy to find, while more strategic/philosophical docs are under two levels. Maybe people will miss them, and I think those are the ones we'd really want them to read.

Comment (mmfred) [reply src/nav/style-guide.yml:68]: I moved the five question and how to organize your docs pages up to the initial section, on writing strategy. The rest is very much about how to do docs things, like our editing processes, etc. It's pretty nuts and bolts about how to docs. I'll create a link on the Intro to the style guide page though.

**Comment by mmfred** [comment]
Waiting on Gatsby fix.

**Review by bradleycamacho — APPROVED** [review]
I tested quite a few of the links and redirects, and they all worked! I assume the rest do too. 

Content wise, it all looks great! My only thought is that maybe we should remove the `For more help` footer from the style guide? Seems a bit irrelevant here.

**Pull request merged**